modules/ROOT/pages/dw-core-annotations.adoc

```diff
@@ -1,12 +1,10 @@
 = Core Annotations (dw::Core)
 
-
-[%header,cols="2,2,4a"]
 |===
 | Annotation | Definition | Description
 
 | AnnotationTarget
-| `@AnnotationTarget(targets: Array<String>)`
+| `@AnnotationTarget&#40;targets: Array<"Function" &#124; "Parameter" &#124; "Variable" &#124; "Import"&#62;&#41;`
```

Comment:
TODO: copyedit source

Reply:
TODO: should all these annotations be customer facing. 

TODO: IF SO, how to use AnnotationTarget. Need example?

Reply:
TODO: fix annotations table output by adding [%header, cols="1,1,1a"] at the top to allow for new lines in cells